lib/ui/screens/calendar_screen.dart

```diff
@@ -297,7 +294,9 @@ class CalendarScrollView extends StatelessWidget {
                 SliverToBoxAdapter(
                   child: TextButton(
```

Comment:
Doesn't need to be a button any more I guess (anyway, as a button its corners when pressed don't align nicely)